spec.html

```diff
@@ -6900,9 +6901,8 @@ <h1>Global Environment Records</h1>
 
         <emu-clause id="sec-global-environment-records-hasbinding-n">
           <h1>HasBinding ( _N_ )</h1>
-          <p>The concrete Environment Record method HasBinding for global Environment Records simply determines if the argument identifier is one of the identifiers bound by the record:</p>
+          <p>The HasBinding concrete method of a global Environment Record _envRec_ takes argument _N_ (a String). It simply determines if the argument identifier is one of the identifiers bound by the record. It performs the following steps when called:</p>
```

Comment:
I don't really like the uses of "simply" I've seen so far. They don't appear to add anything.

Reply:
Yeah, they struck me as a bit odd too. Deleting 3 occurrences.